src/pykeen/pipeline/api.py

```diff
@@ -916,10 +916,12 @@ def pipeline(  # noqa: C901
         evaluator_kwargs = {}
     evaluator_kwargs = dict(evaluator_kwargs)
     if evaluator_kwargs.get('filtered'):
-        all_pos_triples = torch.cat([training.mapped_triples, testing.mapped_triples], dim=0)
-        if validation is not None:
-            all_pos_triples = torch.cat([all_pos_triples, validation.mapped_triples], dim=0)
-        evaluator_kwargs['all_pos_triples'] = all_pos_triples
+        if use_testing_data and validation is not None:
+            logging.info("Validation triples are added to the set of known positive triples which are filtered out"
+                         "during evaluation since we evaluate on the test set. "
+                         "This is the standard approach described by (Bordes et. al, 2013).")
+            evaluator_kwargs['additional_pos_triples'] = validation.mapped_triples
```

Comment:
So, evaluation and training triples are always used by default? Where do the training triples get added?

Reply:
Yes, they are always used by default.
The training and evaluation triples are added in the evaluator itself (as before):
https://github.com/pykeen/pykeen/blob/c61aa83e2b72982047d77ec96ed3af270ec7759c/src/pykeen/evaluation/evaluator.py#L557

Reply:
:+1: